ted/src/main/java/frc/robot/commands/ShooterSetAngleUntilLimitCommand.java

```diff
@@ -0,0 +1,136 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Crescendo - 2024
+// File: ShooterSetAngleUntilLimitCommand.java
+// Intent: Forms a command to update the shooter angle in one direction until the directional limit is reached
+// or until the command scheduler stops its execution (whichever comes first)
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import frc.robot.common.MotorUtils;
+import frc.robot.control.Constants;
+import frc.robot.subsystems.ShooterAngleSubsystem;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command; 
+
+/**
+ * Forms a command to update the shooter angle in one direction until the directional limit is reached
+ * or until the command scheduler stops its execution (whichever comes first)
+ */
+public class ShooterSetAngleUntilLimitCommand extends Command {
+
+  // A block of step functions for time and angle step sizes
+  private static final double firstTimeLimitSeconds = 0.6;
+  private static final double firstStepSizeDegrees = Constants.shooterAngleStickIncrementMagnitude * 2.5;
```

Comment:
why do the step sizes get smaller the longer you run? I would have expected them to get bigger the longer you run.

Reply:
There was a bug on line 34 (at least, a bug that deviated from my original intent - updated now).

We had agreed on a step function type implementation yesterday.  My thought was full exponential steps would be too much at the top-side steps.  It seemed to me with no testing, that step changes that 'less exponential' as things increase were more appropriate.

Step example with the code:
|--|--|--|
|step    | degree step size | linear 2.5x multiplier (hypothetical) |
|step 0 | 0.9                       | 0.9                                                   |
|step 1 | 2.25                     | 2.25                                                 |
|step 2 | 4.50                     | 5.625                                               |
|step 3 | 7.875                   | 14.0625                                           |
|step 4 | 11.8125               | 35.15625                                         |

I'm up to change it however folks want.

Reply:
we decided to go with constant speed after all of that ... removed all of this code